README.md

```diff
@@ -1,2 +1,233 @@
+
+```python
+>>> pyminio_client.isdir('/foo/bar/baz/file_name_1')  # existed file
+False
+>>> pyminio_client.isdir('/foo/bar/baz/directory_name/')  # existed directory
+True
+>>> pyminio_client.isdir('/all/path/wrong/but/directory/')  # not existed directory
+False
+```
+
+### truncate
+Pyminio.truncate will delete all minio's content.
+
+```python
+>>> pyminio_client.truncate()
+```
+
+### rmdir
+Pyminio.rmdir will delete the specified directory. Works like linux's 'rmdir (-r)'.
+
+It will raise a DirectoryNotEmptyError if given directory is not empty, except if the recursive flag is on and then it will delete given directory's path recursively.
+
+This function must get a directory path or it will raise a ValueError.
+
+```python
+>>> pyminio_client.rmdir('/foo/bar/baz/directory_name/')  # empty directory
+>>> pyminio_client.rmdir('/foo/bar/')  # unempty directory
+DirectoryNotEmptyError: can not recursively delete unempty directory
+>>> pyminio_client.rmdir('/foo/bar/', recursive=True)
+```
+
+### rm
+Pyminio.rm works like [rmdir](#rmdir) only that it can delete files too. Works like linux's rm (-r).
+
+```python
+>>> pyminio_client.rm('/foo/bar/baz/file_name')
+```
+
+### cp
+Pyminio.cp will copy one file or directory to given destination. Works like linux's 'cp (-r)'.
+
+This func can only copy recursively when the recursive flag is True. If not, it will raise a ValueError.
+
+#### How will the copy accure? (all directories are copied recursively in this examples)
+| src path   | dst path  | dst exists | new dst      | Explain                                                                |
+| ---------- | --------- | ---------- | ------------ | ---------------------------------------------------------------------- |
+| /foo/bar   | /foo/baz  |    ---     | /foo/baz     | The file's name will be changed from bar to baz.                       |
+| /foo1/bar  | /foo2/    |    True    | /foo/bar     | The file will be copied to '/foo2/'                                    |
+| /foo/bar/  | /foo/     |    True    | /foo/        | The content of '/foo/bar/' will be copied to '/foo/'                   |
+| /foo1/bar/ | /foo2/    |   False    | /foo2/bar/   | '/foo1/bar/' will be copied recursively to '/foo2/bar/'                |
+| /foo1/bar/ | /foo2/baz |    ---     |     ---      | ValueError will be raised in attempting to copy directory in to a file |
+
+```python
+>>> pyminio_client.cp('/foo/bar', '/foo/baz')
+>>> pyminio_client.cp('/foo1/bar', '/foo2/')
+>>> pyminio_client.cp('/foo/bar/', '/foo/', recursive=True)
+>>> pyminio_client.cp('/foo1/bar/', '/foo2/', recursive=True)
+>>> pyminio_client.cp('/foo1/bar/', '/foo2/baz', recursive=True)
+ValueError: can not activate this method from directory to a file.
+```
+
+### mv
+Pyminio.mv works like [cp](#cp) only that it remove the source after complete the transfare. Works like linux's 'mv (-r)'.
```

Comment:
Pyminio.mv -> the `mv` method
*only that it removes the source after the transfer has been completed
No `-r` in `mv`